1.0/elements/docs-menu.html

```diff
@@ -26,20 +26,18 @@
     </core-submenu>
 
     <core-submenu label="Get started" icon="social:school" class="guide">
-      <core-item label="What is Polymer?"><a href="/{{versionPrefix}}/docs/start/what-is-polymer.html"></a></core-item>      
-      <core-item label="Quick tour of Polymer"><a href="/{{versionPrefix}}/docs/start/quick-tour.html"></a></core-item>  
+      <core-item label="What is Polymer?"><a href="/{{versionPrefix}}/docs/start/what-is-polymer.html"></a></core-item>
+      <core-item label="Quick tour of Polymer"><a href="/{{versionPrefix}}/docs/start/quick-tour.html"></a></core-item>
       <core-item label="Get the code"><a href="/{{versionPrefix}}/docs/start/getting-the-code.html"></a></core-item>
-      <core-item label="Reusable elements"><a href="/{{versionPrefix}}/docs/start/reusableelements.html"></a></core-item>   
+      <core-item label="Reusable elements"><a href="/{{versionPrefix}}/docs/start/reusableelements.html"></a></core-item>
     </core-submenu>
 
     <core-submenu label="Developer guide" icon="description" class="guide">
       <core-item label="Feature overview"><a href="/{{versionPrefix}}/docs/devguide/feature-overview.html"></a></core-item>
       <core-item label="Registration and lifecycle"><a href="/{{versionPrefix}}/docs/devguide/registering-elements.html"></a></core-item>
       <core-item label="Declared properties"><a href="/{{versionPrefix}}/docs/devguide/properties.html"></a></core-item>
-      <core-submenu label="Local DOM" icon="hardware:keyboard-arrow-down" class="guide">
-        <core-item label="Basics and API"><a href="/{{versionPrefix}}/docs/devguide/local-dom.html"></a></core-item>
-        <core-item label="Styling"><a href="/{{versionPrefix}}/docs/devguide/styling.html"></a></core-item>
-      </core-submenu>
+      <core-item label="Styling"><a href="/{{versionPrefix}}/docs/devguide/styling.html"></a></core-item>
```

Comment:
I think styling should follow local DOM -- even though it's not exclusively related to local DOM, it refers to a bunch of local DOM concepts.
Also, can you just cut "Local DOM basics and API" to "Local DOM"? If there's only one section on local DOM, there's no need to be verbose here.